src/editor/map/context_manager.hpp

```diff
@@ -25,11 +25,11 @@ class game_config_view;
 
 namespace editor
 {
+using context_ptr = std::unique_ptr<map_context>;
```

Comment:
Remove this alias and just use the unique_ptr type directly. And replace the instances of `new map_context` in `context_manager` with `std::make_unique<map_context>`.

Reply:
moved the alias back to where it originally was. Faced segfaults while replacing the `new map_context`-s.

Reply:
Suggest removing the alias in the class as well. It makes it clearer to have consistent usage. As for the segfaults, that really shouldn’t happen… Do you know exactly where it was crashing?

Reply:
Fixed in d9b695289fc80de68f97f224bbc41fa4cab8b4cf